cmd/release/main.go

```diff
@@ -288,7 +320,7 @@ func (g git) tag(version string) error {
 		// assume this means it doesn't exist
 		cmd = exec.Command("git", "tag", "-s", "-m", fmt.Sprintf("A release of %q for version %q", g.repository, version), version)
 		out, err2 := cmd.CombinedOutput()
-		if err != nil {
+		if err2 != nil {
 			fmt.Println(string(out))
 		}
 		return err2
```

Comment:
Shouldn't this return `err`, since `err2` here is not guaranteed to be non-nil?

Reply:
no, the problem here is go1.11 doesn't let you get exit codes. The assumption here is that if `err` is not nil, then it's erring because the tag exists which is a known case and we can discard the error.

Therefore err2 is the only interesting error at this point.

This will look better when we migrate to go 1.12 that allows me to check exit codes cleanly

Reply:
Gotcha, I think it might make sense to be more explicit about it then, otherwise it might cause others confusion down the line as well.

Reply:
there is a comment *just* out of context of what the diff shows -- does that comment look like it explains what I've described above or do you think I need more words?